ios/Exponent/Kernel/ReactAppManager/EXReactAppManager.m

```diff
@@ -264,10 +263,6 @@ - (void)loadSourceForBridge:(RCTBridge *)bridge withBlock:(RCTSourceLoadBlock)lo
 
 - (NSArray *)extraModulesForBridge:(RCTBridge *)bridge
 {
-  // we allow the vanilla RN dev menu in some circumstances.
-  BOOL isDetached = [EXEnvironment sharedEnvironment].isDetached;
```

Comment:
it's notable that this will impact expokit on sdk 37 - if we do follow through with stopping support on it then this won't be an issue

Reply:
That's right, I've reverted this change.